msteams-platform/concepts/deploy-and-publish/appsource/prepare/submission-checklist.md

```diff
@@ -1,90 +1,53 @@
-* **Self test your app:** Test your app using the [Manifest validation tool](#teams-app-validation-tool).
-* **App detail page:** Your app must align with the  [App detail page checklist](detail-page-checklist.md).
-* **Tips and frequently failed cases:** Pay extra attention to the listed [Tips and frequently failed cases](frequently-failed-cases.md)  to improve your app submission and approval time.
-* **App manifest:** Check your app manifest against the [App manifest checklist](app-manifest-checklist.md).
-* **Testing and debugging:** Make certain that you have fully [tested and debugged your app](../../../build-and-test/debug.md).
-* **Testing notes:** Include your [test notes for validation](#test-notes-for-validation)
-* **Privacy policies:** Ensure your [privacy policy, terms of use and support URLs](#privacy-policy-terms-of-use-and-support-urls) follow our guidelines.
+Before you submit your app to [Partner Center](/office/dev/store/use-partner-center-to-submit-to-appsource), read the following instructions.
 
-Once you have completed all of the above requirements, submit your package to AppSource through [Partner Center](/office/dev/store/use-partner-center-to-submit-to-appsource).
+## Validate your app manifest
 
-## Teams App Validation Tool
+While your app may be working in a test environment, you should check your app manifest again to avoid running into issues during the submission process.
 
-The app validation tool consists of an [app validator](#teams-app-validator) and a [preliminary checklist](#preliminary-checklist). The tool replicates the same test cases used by [AppSource](/office/dev/store/submit-to-appsource-via-partner-center) to evaluate your app submission. Therefore,  it's crucial to pass all the test cases prior to submitting your solution to AppSource for approval.The tool can be found in several areas within the Teams platform:
+The Microsoft Teams App Validator tool helps you identify and fix issues before submitting to Partner Center. The tool automatically checks your manifest against the same test cases used during store validation.
 
-> [!div class="checklist"]
->
-> * [**App Validator homepage**](https://dev.teams.microsoft.com/appvalidation.html)
-> * [**Teams Visual Studio Code toolkit**](/toolkit/visual-studio-code-overview.md)
-> * [**App Studio**](../../../build-and-test/app-studio-overview.md)
+1. Go to the [Microsoft Teams App Validator tool](https://dev.teams.microsoft.com/appvalidation.html). (Note: The tool is also available in [App Studio](../../../build-and-test/app-studio-overview.md).)
+1. Upload your manifest to run the automated tests.
+1. Go to the **Preliminary checklist** and review the list of test cases that are difficult to automate.
+1. If present, [fix issues with your manifest](~/resources/schema/manifest-schema.md) or app in general.
```

Comment:
@heath-hamilton - Include snapshot for Validator tool & Prem
![image](https://user-images.githubusercontent.com/78354231/109552370-b2310580-7af7-11eb-9730-24fe62f460f8.png)
![image](https://user-images.githubusercontent.com/78354231/109552426-c2e17b80-7af7-11eb-9b83-1ac7c6243886.png)